frontend/src/component/playground/Playground/PlaygroundResultsTable/FeatureResultInfoPopoverCell/FeatureResultInfoPopoverCell.tsx

```diff
@@ -65,32 +60,96 @@ export const FeatureResultInfoPopoverCell = ({
                     horizontal: 'left',
                 }}
             >
-                {useNewStrategyDesign ? (
-                    <>
-                        <FeatureDetails
-                            feature={feature}
-                            input={input}
-                            onClose={() => setOpen(false)}
-                        />
-                        <PlaygroundResultFeatureStrategyList
-                            feature={feature}
-                            input={input}
-                        />
-                    </>
-                ) : (
-                    <>
-                        <LegacyFeatureDetails
-                            feature={feature}
-                            input={input}
-                            onClose={() => setOpen(false)}
-                        />
-                        <LegacyPlaygroundResultFeatureStrategyList
-                            feature={feature}
-                            input={input}
-                        />
-                    </>
-                )}
+                <LegacyFeatureDetails
+                    feature={feature}
+                    input={input}
+                    onClose={() => setOpen(false)}
+                />
+                <LegacyPlaygroundResultFeatureStrategyList
+                    feature={feature}
+                    input={input}
+                />
+            </Popover>
+        </FeatureResultPopoverWrapper>
+    );
+};
+
+const DetailsPadding = styled('div')(({ theme }) => ({
+    paddingInline: `var(--popover-inline-padding, ${theme.spacing(4)})`,
+    paddingTop: theme.spacing(2.5),
+}));
+
+export const NewFeatureResultInfoPopoverCell = ({
+    feature,
+    input,
+}: FeatureResultInfoPopoverCellProps) => {
+    const [open, setOpen] = useState(false);
+    const ref = useRef(null);
+
+    const togglePopover = () => {
+        setOpen(!open);
+    };
+
+    return (
+        <FeatureResultPopoverWrapper>
+            <IconButton onClick={togglePopover}>
+                <InfoOutlined ref={ref} />
+            </IconButton>
+            <Popover
+                open={open}
+                onClose={() => setOpen(false)}
+                anchorEl={ref.current}
+                PaperProps={{
+                    sx: (theme) => ({
+                        '--popover-inline-padding': theme.spacing(4),
```

Comment:
We can't set the padding here because the strategy separators should expand all the way to the edges. Instead, we can set a CSS variable and access it in the downstream components that need it.